src/components/ReportActionItem/TaskView.js

```diff
@@ -0,0 +1,148 @@
+                    <>
+                        <Text style={styles.taskTitleDescription}>Title</Text>
+                        <View style={[styles.flexRow, styles.alignItemsTop, styles.flex1]}>
+                            <Checkbox
+                                onPress={() => (isCompleted ? Task.reopenTask(props.report.reportID, taskTitle) : Task.completeTask(props.report.reportID, taskTitle))}
+                                isChecked={isCompleted}
+                                style={styles.taskMenuItemCheckbox}
+                                containerSize={24}
+                                containerBorderRadius={8}
+                                caretSize={16}
+                                accessibilityLabel={taskTitle || props.translate('task.task')}
+                                disabled={isCanceled}
+                            />
+                            <View style={[styles.flexRow, styles.flex1]}>
+                                <Text
+                                    numberOfLines={3}
+                                    style={styles.taskTitleMenuItem}
+                                >
+                                    {taskTitle}
+                                </Text>
+                            </View>
+                            {isOpen && (
+                                <View style={styles.taskRightIconContainer}>
+                                    <Icon
+                                        additionalStyles={[styles.alignItemsCenter]}
+                                        src={Expensicons.ArrowRight}
+                                        fill={StyleUtils.getIconFillColor(getButtonState(hovered, pressed, false, !isOpen))}
+                                    />
+                                </View>
+                            )}
+                        </View>
+                    </>
+                )}
+            </PressableWithSecondaryInteraction>
+            <MenuItemWithTopDescription
+                description={props.translate('task.description')}
+                title={props.report.description || ''}
+                onPress={() => Navigation.navigate(ROUTES.getTaskReportDescriptionRoute(props.report.reportID))}
+                shouldShowRightIcon={isOpen}
+                disabled={!isOpen}
+                wrapperStyle={[styles.pv2]}
+                numberOfLinesTitle={3}
+                shouldGreyOutWhenDisabled={false}
+            />
+            {props.report.managerID ? (
+                <MenuItem
+                    label={props.translate('task.assignee')}
+                    title={ReportUtils.getDisplayNameForParticipant(props.report.managerID)}
+                    icon={UserUtils.getAvatar(
+                        PersonalDetailsUtils.getPersonalDetailsByIDs([props.report.managerID], props.currentUserPersonalDetails.accountID)[0].avatar,
+                        props.report.managerID,
+                    )}
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/22657. `PersonalDetailsUtils.getPersonalDetailsByIDs` could return an empty array e.g. for users that you have not interacted with before (no personaldetails). Then we will be trying to access property `avatar` of `undefined`.